docs/archive-data.md

```diff
@@ -0,0 +1,314 @@
+| `archival.history.provider.filestore.fileMode` | This specifies the file permissions of the archived files. We recommend using the default value of `"0666"` to avoid read/write issues. |
+| `archival.history.provider.filestore.dirMode`  |  This specifies the directory permissions of the archive directory. we recommend using the default value of `"0766"` to avoid read/write issues. |
+| `namespaceDefaults.archival.history.state`     | Acceptable values are `"enabled"`, `"disabled"`. This sets the default state of the Archival feature whenever a new Namespace is created without specifying the Archival state. |
+| `namespaceDefaults.archival.history.URI`       | The value must be a URI of the file store location and match a schema that correlates to a provider. |
+
+Consider the following example:
+
+```yaml
+# Cluster level Archival config
+archival:
+  # Event History configuration
+  history:
+    # State at the cluster level
+    state: "enabled"
+    # Read permissions
+    enableRead: true
+    # Available providers
+    provider:
+      # Local filestore provider  
+      filestore:
+        # File permissions
+        fileMode: "0666"
+        # Directory permissions
+        dirMode: "0766"
+      # Google Cloud provider  
+      gstorage:
+        # Google Cloud credentials file
+        credentialsPath: "/tmp/gcloud/keyfile.json"
+
+# Default values for a Namespace if none are provided at creation
+namespaceDefaults:
+  # Archival defaults
+  archival:
+    # Event History defaults
+    history:
+      # Default state
+      state: "enabled"
+      # Default URI
+      URI: "file:///tmp/temporal_archival/development"
+```
+
+In the configuration above we see that:
+
+1. Archival is enabled at the cluster level.
+2. Namespaces can use either the local filestore provider or the Google Cloud provider.
+3. New Namespaces will default to the local provider, 
+
+:::tip
+
+**Want to disable Archival?**
+
+You can disable Archival by setting `archival.history.state` and `namespaceDefaults.archival.history.state` to `"disabled"`.
+
+Example:
+
+```yaml
+archival:
+  history:
+    state: "disabled"
+
+namespaceDefaults:
+  archival:
+    history:
+      state: "disabled"
+```
+
+:::
+
+### Namespace creation
+
+The Archival URI is set when a Namespace is created. If you do not specify the URI when creating a Namespace, it will use the value of `defaultNamespace.archival.history.URI` from the `config/development.yaml` file. Once set, the URI can not be changed. Each Namespace supports a single Archival URI, but each Namespace can use a different URI (archiver).
+
+At the cluster level, `archival.history.state` must be set to `"enabled"` for Archival to work at the Namespace level for Even Histories. Once enabled at the cluster level, a Namespace can safely switch between `enabled` and `disabled` states.
```

Comment:
"Even Histories"

Also reword,

"To enable Namespace archival, `archival.history.state` must be set to `"enabled"` at a cluster level. Only then a can Namespaces safely switch between `enabled` and `disabled` states.: